docs/templates/PROJECTS.md

```diff
@@ -1,109 +0,0 @@
-- [List of available projects](#list-of-available-projects)
```

Comment:
I guess you meant to rename PROJECTS to REUSE, but instead have deleted it ?

Reply:
I did rename PROJECTS as REUSE -- and I thought I corrected all mentions. Can you point out the error more clearly?